CHANGELOG.md

```diff
@@ -14,6 +32,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [3.1.37] - 2021-01-07
 
 ### Changed
+- Dropped support for PHP versions lower than PHP7.1 (and disabled unit tests for 7.1) 
```

Comment:
This line looks conflict with the below line `Travis unit tests now run for all php versions >= 5.3, including php8`.
I guess that it is for smarty4 ;)